bpf/bpf_host.c

```diff
@@ -1126,9 +1123,13 @@ int to_host(struct __ctx_buff *ctx)
 #endif /* ENABLE_HOST_FIREWALL */
 
 out:
-	if (IS_ERR(ret))
+	if (IS_ERR(ret)) {
 		return send_drop_notify_error(ctx, srcID, ret, CTX_ACT_DROP,
 					      METRIC_INGRESS);
+	} else if(!traced){
```

Comment:
I don't think the curly brackets (`{}`) are needed for the if/else here.

Reply:
Ok! I will simplify this and squash the commits